src/components/Button/Button.tsx

```diff
@@ -0,0 +1,24 @@
+import cn from 'classnames';
+
+import styles from './Button.module.css';
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  priority: string;
+  children: React.ReactNode;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
`React.ButtonHTMLAttributes<HTMLButtonElement>` includes types for `children` and `onClick`. There is no need to define these types again